draft-ietf-quic-http.md

```diff
@@ -2316,20 +2320,26 @@ SETTINGS_HEADER_TABLE_SIZE:
 
 SETTINGS_ENABLE_PUSH:
 : This is removed in favor of the MAX_PUSH_ID which provides a more granular
-  control over server push.
+  control over server push.  Specifying a setting with the identifier 0x2
```

Comment:
Could we instead phrase these as normative? For example: `Endpoints MUST NOT send a setting with identifier 0x2 (corresponding to the SETTINGS_ENABLE_PUSH parameter) in the HTTP/3 SETTINGS frame.`

Reply:
Nothing in this appendix is normative, by design.  The normative requirement is in the body of the document, and this appendix simply notes that there is a prohibition.  I could add a reference to the prohibition, if that would help.

Reply:
A reference to the prohibition works for me, I just want to make sure we have text somewhere clearly stating that endpoints MUST NOT send those